cypress/e2e/case-list/filter-cases/filterChips.cy.ts

```diff
@@ -322,13 +302,11 @@ describe("Case list", () => {
         cy.get(".moj-filter-tags").contains("Triggers").should("exist")
 
         // Clears filter chip and check the checkbox is deselected
-        cy.get(".moj-filter-tags").contains("Clear filters").click()
-        cy.get("#filter-button").click()
+        cy.get('a[href*="/bichard?keywords="]').contains("Clear filters").click()
```

Comment:
Do we have a better identifier we can use here? 🤔

Reply:
done